source/module_elecstate/module_charge/charge_mixing.cpp

```diff
@@ -123,23 +124,27 @@ void Charge_Mixing::init_mixing()
             this->mixing->init_mixing_data(this->rho_mdata,
                                         this->rhopw->npw * 2,
                                         sizeof(std::complex<double>));
+            ModuleBase::Memory::record("Charge_Mixing::init_mixing_data",sizeof(std::complex<double>)*this->mixing->data_ndim*this->rhopw->npw * 2);
```

Comment:
this line seems should be put into the function "init_mixing_data"

Reply:
OK, I will modify it

Reply:
![image](https://github.com/deepmodeling/abacus-develop/assets/73126527/27f62711-f9c5-4d5e-80e4-fa3ab812159b)

Reply:
But I'm a little confused. If put this line into function `init_mixing_data`：
![image](https://github.com/deepmodeling/abacus-develop/assets/73126527/0e5cdde9-af12-46df-b283-0938421b086a)
we cannot record the full name， such as `Charge_Mixing::init_mixing_data`，and it may not be easy to locate the specific location of malloc memory.